File: ABC/ABC367/A/main.cc
#include <iostream>

int main() {
  int a, b, c;
  std::cin >> a >> b >> c;
  if (a < b) a += 24;
  if (c < b) c += 24;
  std::cout << ((a - b) * (c - a) < 0 ? "Yes" : "No") << "\n";
  return 0;
}

File: ABC/ABC367/A/main.go
package main

import "fmt"

func main() {
	var a, b, c int
	fmt.Scan(&a, &b, &c)

	if a < b {
		a += 24
	}
	if c < b {
		c += 24
	}

	if (a - b) * (c - a) < 0 {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}